packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -361,6 +362,7 @@ export default function MainFeedLayout({
           directlyOpenedTab="Manage tags"
         />
       )}
+      <FeedFitlersModal isOpen /> {/* temporary for development */}
```

Comment:
We just have to be very careful with this.
Possibly in future not even commit it, but use it locally, or perhaps adding a TODO block?

More a remark for discussion at gathering.